client/src/Components/Projects.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import '../Style/Projects.css'
+
+export default function Projects(props) {
+
+    return (
+        <div id="project-head" >
+            <br></br>
+            <h1>Check out what I've made so far...</h1>
+
+            <div id="how-to">
+                <p>Here are a few of my favorite development creations to date...<br></br>click on any title to see the git repository.</p>
+            </div>
+            
+            <div id="project-body">
+                {/* all links open to git repositories in separate window */}
+                <a className="git-link" href="https://github.com/BurlingtonCodeAcademy/yelpington-JenaDurell" target="_blank">Yelpington!</a>
+                <div className="project-info">
+                    <p>Yelpington is an app loosely based on Yelp, where the user is able to look at a map, click on a restaurant location, and go to both the Yelpington page for that restaurant as well as the website published by the restaurant. Yelpington is written using Javascript, HTML, CSS, leaflet maps and JQuery. </p>
+                </div>
```

Comment:
Including screenshots of your projects into here could be a good thing to add.

This goes for each of your projects